apps/openmw/mwworld/projectilemanager.cpp

```diff
@@ -444,11 +461,14 @@ namespace MWWorld
                 MWBase::Environment::get().getWorld()->explodeSpell(pos, it->mEffects, caster, result.mHitObject,
                                                                     ESM::RT_Target, it->mSpellId, it->mSourceName);
 
+                if (result.mProjectileId >= 0)
+                    markToRemove(result.mProjectileId);
+
                 MWBase::SoundManager *sndMgr = MWBase::Environment::get().getSoundManager();
                 for (size_t soundIter = 0; soundIter != it->mSounds.size(); soundIter++)
                     sndMgr->stopSound(it->mSounds.at(soundIter));
 
-                mParent->removeChild(it->mNode);
+                cleanupMagicBolt(*it);
```

Comment:
cleanupMagicBolt() already handles stopping sounds, so the code above should be removed.